### ISSUE TITLE
pretrained

### ISSUE THREAD

**Issue by jeansely** [CharlesShang/FastMaskRCNN#130]
Hi,
where can find  pre-trained model of Mask-RCNN?

**Comment by LBJ66** [comment]
wget  http://download.tensorflow.org/models/resnet_v1_50_2016_08_28.tar.gz
then you can get the pretrained resNet50 model

**Comment by jeansely** [comment]
thanks for response, but i am looking for trained weights in mask-rcnn (weights in final train).
because i can not train this project on CPU and windows os .

**Comment by jackytu256** [comment]
Hi jeasnsely,

  I think that, from my perspective, no one gets a good result of training mask-rcnn model at the moment

**Comment by AndrewKemendo** [comment]
Try this one: #86

**Comment by ya2** [comment]
Where should 1 place the pre-trained .meta .index and .data files to run the model ?

**Comment by AndrewKemendo** [comment]
So long as you point your test system to the right path it won't matter.

That said, all the ways I have tried require a checkpoint file, which is
missing in this case.

See:
http://cv-tricks.com/tensorflow-tutorial/save-restore-tensorflow-models-quick-complete-tutorial/

**Comment by ya2** [comment]
What should we do if we don't have a checkpoint file ?

(I'm a complete noobie here, sorry.)
